example/provider-local/base/cloudprofile.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   - name: local
   kubernetes:
     versions:
+    - version: 1.23.6
```

Comment:
The first 4 commits are not required. I guess you mistakenly added it.

Reply:
Ah yes, this is a mistake. Thanks for pointing it out.